app/controllers/rules_of_origin/steps_controller.rb

```diff
@@ -13,10 +13,10 @@ class StepsController < ApplicationController
 
     before_action :check_session_data,
                   except: :index,
-                  if: -> { params[:id] != wizard_class.steps.first.key }
+                  if: -> { params[:id] != 'start' }
 
     def show
-      if params[:id] == wizard_class.steps.first.key
+      if params[:id] == 'start'
```

Comment:
Do we have constant for these?

Reply:
@willfish Not really though if we're going down that route I'm probably best just grabbing the first key as I was before?